src/ng/rootScope.js

```diff
@@ -892,6 +893,58 @@ function $RootScopeProvider() {
         $browser.$$checkUrlChange();
       },
 
+      /**
+       * @ngdoc method
+       * @name $rootScope.Scope#$suspend
+       * @kind function
+       *
+       * @description
+       * Suspend watchers of this scope subtree so that they will not be invoked during digest.
+       *
+       * This can be used to optimize your application when you know that running those watchers
+       * is redundant.
+       *
+       * **Warning**
+       *
+       * Suspending scopes from the digest cycle can have unwanted and difficult to debug results.
+       * Only use this approach if you are confident that you know what you are doing and have
+       * ample tests to ensure that bindings get updated as you expect.
+       *
+       * Some of the things to consider are:
```

Comment:
A few other things:

-  isolate scopes are also suspended (as they are still children of the parent and not the rootScope)

- resolved promises e.g. from explicit $q deferreds and http calls are not bound to scopes and will trigger a global digest if you e.g. execute http calls from a component which has its scope suspended.